pkg/obsservice/obslib/migrations/sqlmigrations/0001_init.sql

```diff
@@ -1,5 +1,13 @@
 -- +goose Up
-CREATE TABLE events();
+CREATE TABLE cluster_events(
+  timestamp TIMESTAMP NOT NULL,
+  id BYTES NOT NULL DEFAULT uuid_v4(),
+  cluster_id BYTES NOT NULL,
+  node_id INT NOT NULL,
```

Comment:
Are you interested in calling this `instance_id`? Should this have a `tenant_id`?